### DISCUSSION TITLE
希望DDE的任务栏能够取消图标合并状态

### DISCUSSION THREAD

**Discussion by LinuxResearcher** [linuxdeepin/developer-center#10422] [Features Request & Ideas | 特性请求 & 头脑风暴]
现在DDE的任务栏图标是始终合并的，我看起来非常地不爽，用起来非常地别扭。

Windows可以设置任务栏图标取消合并状态，gnome桌面也有windows-list插件，kde也能设置类似的效果。

作为一个习惯用XP系统任务栏的人，相信有很多用户也有类似的习惯。所以，希望DDE任务栏能像Windows 10那样能够设置取消始终合并任务栏图标。

**Comment by BLumia** [comment]
我这边会把你的建议转给产品，不过我也建议你通过deepin内置的深度之家应用进行下反馈，这样那边可以更好的跟进相关的需求。

**Comment by LinuxResearcher** [comment]
好的，谢谢！



---原始邮件---
发件人: "Gary ***@***.***&gt;
发送时间: 2024年8月17日(周六) 晚上10:57
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [linuxdeepin/developer-center] 希望DDE的任务栏能够取消图标合并状态 (Discussion #10422)




 
我这边会把你的建议转给产品，不过我也建议你通过deepin内置的深度之家应用进行下反馈，这样那边可以更好的跟进相关的需求。
